meshtastic/tests/test_node.py

```
@@ -341,6 +341,77 @@ def test_shutdown(caplog: LogCaptureFixture) -> None:
+) -> None:
+    """factoryReset(full=False) should set config reset flag as int and skip callback for local node."""
+    iface = autospec_local_node_iface(MeshInterface)
+    anode = Node(iface, "!12345678", noProto=True)
+    iface.localNode = anode
+    anode.ensureSessionKey = MagicMock()  # type: ignore[method-assign]
+    captured: dict[str, object] = {}
+    sent_packet = mesh_pb2.MeshPacket()
+
+    def _fake_send_admin(
+        msg: admin_pb2.AdminMessage,
+        wantResponse: bool = False,
```

Ruff (0.15.2)

[error] Boolean-typed positional argument in function definition

(FBT001)